internal-services/manager/production/manager-prod-rh02.yaml

```diff
@@ -0,0 +1,106 @@
+      # affinity:
+      #   nodeAffinity:
+      #     requiredDuringSchedulingIgnoredDuringExecution:
+      #       nodeSelectorTerms:
+      #         - matchExpressions:
+      #           - key: kubernetes.io/arch
+      #             operator: In
+      #             values:
+      #               - amd64
+      #               - arm64
+      #               - ppc64le
+      #               - s390x
+      #           - key: kubernetes.io/os
+      #             operator: In
+      #             values:
+      #               - linux
+      securityContext:
+        runAsNonRoot: true
+        # TODO(user): For common cases that do not require escalating privileges
+        # it is recommended to ensure that all your Pods/Containers are restrictive.
+        # More info: https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted
+        # Please uncomment the following code if your project does NOT have to work on old Kubernetes
+        # versions < 1.19 or on vendors versions which do NOT support this field by default (i.e. Openshift < 4.11 ).
+        # seccompProfile:
+        #   type: RuntimeDefault
+      containers:
+        - command:
+            - /manager
+          args:
+            - --leader-elect
+            - --remote-cluster-config-file
+            - /mnt/internal-services/remote-client-config/kubeconfig
+            - --leader-election-id
+            - "prod-rh02"
+          image: quay.io/konflux-ci/internal-services:f625b58a23df1a02e981519133e44b28aff45e20
+          name: manager
+          securityContext:
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+                - "ALL"
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8081
+            initialDelaySeconds: 15
+            periodSeconds: 20
+          readinessProbe:
+            httpGet:
+              path: /readyz
+              port: 8081
+            initialDelaySeconds: 5
+            periodSeconds: 10
+          # TODO(user): Configure the resources accordingly based on the project requirements.
+          # More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+          resources:
+            limits:
+              cpu: 500m
+              memory: 512Mi
+            requests:
+              cpu: 10m
+              memory: 512Mi
+          env:
+            - name: SERVICE_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+          volumeMounts:
+            - mountPath: /mnt/internal-services/remote-client-config
+              name: remote-client-config
+              readOnly: true
+      serviceAccountName: controller-manager
+      terminationGracePeriodSeconds: 10
+      volumes:
+        - name: remote-client-config
+          secret:
+            secretName: remote-client-config-prod-rh02
```

Comment:
I think this will likely fail if the secret doesn't exist, so it should be added first, no? Not sure where though. Could it be that it's just done manually on the cluster? Because I don't see remote-client-config-prod-p02 defined anywhere either.

Reply:
Yes, it has to be defined in Vault. That's why I set it as a draft.

Reply:
Sure, in Vault, but having it in Vault doesn't automatically mean it gets deployed. Maybe there is a wildcard somewhere to deploy all such secrets? But I don't see it anywhere.

Reply:
You map it in app-interface: https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/137976

Reply:
Sorry, somehow I thought this was about app-infra and Konflux deployments :) Makes sense.